ansible/roles/base/tasks/install.yml

```diff
@@ -0,0 +1,29 @@
+- name: install packages
+  become: yes
+  yum:
+    name:
+    - docker
+    - git
+    state: present
+  tags:
+    - packages
+
+- name: add docker group
+  become: yes
+  group:
+    name: docker
+    state: present
+
+- name: add user to docker group
+  become: yes
+  user:
+    name: "{{ user }}"
+    groups: docker
+    append: yes
+
+- name: start docker
+  become: yes
+  systemd:
+    name: docker
+    enabled: yes
+    state: restarted
```

Comment:
why not started?
```suggestion
    state: started
```

Reply:
it seems the addition of $USER to the docker group does not always take without restarting, but maybe my env was dirty so I'll check again

Reply:
Why was docker started before this task?